estuary_updater/handlers/freshmaker.py

```diff
@@ -79,30 +80,30 @@ def build_state_handler(self, msg):
         """
         build_info = msg['body']['msg']
         event_id = msg['body']['msg']['event_id']
+        freshmaker_build = self.create_or_update_freshmaker_build(build_info, event_id)
         build = self.create_or_update_build(build_info, event_id)
+        event = None
         if build:
```

Comment:
Since `build` will only get created when the Freshmaker build is in the done state, this conditional should change so that `event` is set even when `build` is `None`. This is because we want the `freshmaker_build` node to have a relationship with the `event` while it's in progress.